core/app/assets/js/layout/page.js

```diff
@@ -0,0 +1,58 @@
+import Header from './header';
+import Sidebar from './sidebar';
+import Content from './content';
+import Footer from './footer';
+
+export default function Page( props ) {
+	const getSideBar = () => (
```

Comment:
```suggestion
	const Sidebar = () => (
```

Reply:
It's a problem because we import sidebar and footer components in this component, so the names clash.
I used `AppSidebar` and `AppFooter` as var names. 
Let me know if you have a different idea